css/style.css

```diff
@@ -0,0 +1,1381 @@
+  }
+
+  .indexCo div:nth-child(3) span {
+    font-size: 12px;
+    color: #555;
+    margin-left: 5px;
+  }
+
+  .indexCo div:nth-child(4) {
+    background: #fffaf8;
+    padding: 16px;
+    border-top: 1px solid rgba(255, 68, 34, 0.25);
+  }
+
+  .indexCo div:nth-child(4) a {
+    font-size: 13px;
+    color: #cab5b5;
+  }
+
+  .latest-news h2 a:hover {
+    color: black;
+  }
+
+  .insights li:nth-child(1) h2 a {
+    font-size: 40px;
+  }
+}
+
+@media screen and (max-device-width: 1023px) and (min-device-width: 767px) {
+  .navbar-mobile ul,
+  .second,
+  .transform li,
+  .insights h5 {
+    display: none;
+  }
+
+  header {
+    position: fixed;
+    width: 100%;
+  }
+
+  .navbar-mobile {
+    display: flex;
+    justify-content: space-between;
+    height: 48px;
+    position: absolute;
+    grid-area: a;
+    background: transparent;
+  }
+
+  .black-navbar {
+    grid-area: b;
+    font-size: 11px;
+    margin-left: 106px;
+    justify-content: space-between;
+  }
+
+  .white-navbar {
+    grid-area: c;
+  }
+
+  .spacer {
+    grid-area: d;
+    margin: 0;
+  }
+
+  .navbar-mobile a {
+    width: 106px;
+    height: 80px;
+    background: linear-gradient(to bottom right, #c09, #f42 85%);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .navbar-mobile img {
+    display: block;
+    height: 16px;
+    position: relative;
+    width: 66px;
+  }
+
+  .insights {
+    margin-top: 106px;
+  }
+
+  .insights li {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .insights li img {
+    display: none;
+  }
+
+  .ind-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .index-right {
+    display: flex;
+    flex-direction: row;
+    background-color: white;
+    justify-content: space-evenly;
+    width: 330px;
+    padding: 0;
+    margin: 0;
+  }
+
+  .latest-news ul {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    width: 100%;
+    grid-gap: 5px;
+  }
+
+  .latest-news li {
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    margin-right: 10px;
+    min-height: 220px;
+    min-width: 230px;
+  }
+
+  .latest-news li span {
+    width: 100%;
+    min-height: 135px;
+    display: block;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .latest-news li div {
+    font-size: 14px;
+    margin-left: 0;
+  }
+
+  .latest-news p,
+  .latest-news p a {
+    margin-top: 10px;
+    color: rgb(80, 80, 80);
+    font-size: 12px;
+  }
+
+  .main-articles {
+    display: grid;
+    grid-gap: 1%;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .white-navbar ul:nth-child(1) {
+    margin-left: 106px;
+  }
+
+  .transform li:nth-child(1) {
+    display: block;
+  }
+
+  .indexCo ul li:last-child {
+    display: none;
+  }
```

Comment:
This selector has the same style definition of selectors on lines 911 and 974. Group selectors, or create a new class.

Reply:
I am sorry, but this is not possible because the linters will provide that:
146:1  ✖  Expected selector ".navbar-mobile img"    no-descending-specificity
            to come before selector ".insights li                              
            img"                                                               
  966:3  ✖  Expected selector ".insights li" to come  no-descending-specificity
            before selector ".indexCo ul                                       
            li:last-child"                                                     
  999:3  ✖  Expected selector ".latest-news li" to    no-descending-specificity
            come before selector ".indexCo ul                                  
            li:last-child"                                                     
 1041:3  ✖  Expected selector ".transform             no-descending-specificity
            li:nth-child(1)" to come before selector                           
            ".indexCo ul li:last-child"

Reply:
There is a way, you only need to create a new class.